### PR TITLE
big changes in the ontology of PDDL, this will affect the prolog part…

### PR DESCRIPTION
**Pull request by kicacacko**
… for generating PDDL

### CODE REVIEW

File: planner_reasoner/rdf/base/pp_oslc_shapes.ttl [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix oslc: <http://open-services.net/ns/core#> . #oslc core
+@prefix sh: <http://www.w3.org/ns/shacl#> . # shapes constraint languge SHACL
+@base <http://ontology.cf.ericsson.net/planning_ontology_oslc_shapes> .
+
+
+:ActionShape rdf:type oslc:ResourceShape ;
+            oslc:describes :Action ;
+            oslc:property [a oslc:Property ;
```
Comment (likelion) [inline planner_reasoner/rdf/base/pp_oslc_shapes.ttl]: `oslc:name` is a mandatory field for every property that should contain second part of property's Prefixed Name.
For example here it should be: `oslc:name "hasPrecondition"^^xsd:string`.

File: planner_reasoner/lib/planner_reasoner.pl [hunk 2]
```diff
@@ -87,6 +87,13 @@
                                         SubSubIndent is SubIndent+10,
                                         format(Out, '~*|(~w', [SubSubIndent,PredName1]),
                                         generate_precondition0(SubPredicateList, SubIndent, Out));
+  (rdf(H1, rdf:type, pp:'And') ->
```
Comment (berezovskyi) [inline planner_reasoner/lib/planner_reasoner.pl:95]: could you please add a comment explaining what `H1` and `T1` stand for, please?

**Review by berezovskyi — APPROVED** [review]

File: planner_reasoner/lib/planner_reasoner.pl (hunk 2 above)
Comment (kicacacko) [reply planner_reasoner/lib/planner_reasoner.pl:95]: Head and Tale of a list, but I will anyway re-implement this part once there is some kind of stable PDDL ontology

**Comment by berezovskyi** [comment]
Looks good to me, do you want me to merge it?

**Pull request merged**